src/components/Rating.vue

```diff
@@ -2,20 +2,20 @@
   <div :class="bem()" v-if="show">
     <slot name="header"></slot>
 
-    <a href="#" @click.default="clear" :class="bem('clear')" v-if="currentValue">
+    <a href="#" @click.prevent="clear" :class="bem('clear')" v-if="currentValue">
```

Comment:
yes, but was lazy enough to not separate the commit :/
default is not a valid "modifier", "default" is.